.gitignore

```diff
@@ -29,3 +29,4 @@
 /yarn.lock
 /drivers
 /config/block/
+/.bashrc
```

Comment:
Not everyone uses bash. Please add this kind of ignore to your global `.gitignore`